sdk/nodejs/runtime/invoke.ts

```diff
@@ -101,27 +101,27 @@ export async function streamInvoke(
         const req = createInvokeRequest(tok, serialized, provider, opts);
 
         // Call `streamInvoke`.
-        const call = monitor.streamInvoke(req, {});
```

Comment:
Did this change b/c of a lint issue w. shadowing?

Reply:
Yes